Type error on x86_32

When trying to compile this crate on a `x86_32` linux machine I get a type error because `nr::{CapSet, CapGet}` are `i64` but `libc::syscall` expects an `i32`.

```
   Compiling caps v0.1.1-alpha.0 (file:///tmp/fail)
error[E0308]: mismatched types
  --> src/base.rs:11:36
   |
11 |     let r = unsafe { libc::syscall(nr::CAPGET, hdr, data) };
   |                                    ^^^^^^^^^^ expected i32, found i64

error[E0308]: mismatched types
  --> src/base.rs:19:36
   |
19 |     let r = unsafe { libc::syscall(nr::CAPSET, hdr, data) };
   |                                    ^^^^^^^^^^ expected i32, found i64

error: aborting due to 2 previous errors

error: Could not compile `caps`.

To learn more, run the command again with --verbose.
```
Running `cargo test` with my patch to get it to compile doesn't produce any errors

--- a/src/nr.rs
+++ b/src/nr.rs
@@ -52,9 +52,9 @@ pub const PR_CAP_AMBIENT_CLEAR_ALL: i32 = 4;
 /* from <unistd.h> */
 
 #[cfg(target_arch = "x86")]
-pub const CAPGET: i64 = 184;
+pub const CAPGET: i32 = 184;
 #[cfg(target_arch = "x86")]
-pub const CAPSET: i64 = 185;
+pub const CAPSET: i32 = 185;
 
 #[cfg(target_arch = "x86_64")]
 pub const CAPGET: i64 = 125;

